.nais/kafka/isfrisktilarbeid-vedtak-fattet.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: kafka.nais.io/v1
 kind: Topic
 metadata:
   annotations:
+    kafka.nais.io/removeDataWhenResourceIsDeleted: "true"
```

Comment:
Topic'et skal slettes i en egen PR - ønsker da at Kafka-topic'et også blir slettet på Aiven (det er ikke tatt i bruk i produksjon ennå)

Reply:
Nice, vet at man også kan kjøre `kubectl delete topic <topic-id>` i kubernetes, men vet ikke om dette er ekvivalent?

Reply:
Ja, men tror man må sette dette flagget for at det faktisk skal bli slettet på Aiven i tillegg.